lib/event_stats.rb

```diff
@@ -110,19 +110,19 @@ def user_comments(user)
 
   def review_stats
     stats = Hash[event.tracks.map do |t|
-      [t.name, track_review_stats(t)]
+      [t.name, track_review_stats(t.id.to_s)]
```

Comment:
If you make that tweak to the in_track scope (just left a comment above), the to_s shouldn't be necessary.